test/build/50_ch-grow.bats

```diff
@@ -31,6 +31,29 @@ setup () {
     [[ $output = *'verbose level: 1'* ]]
 }
 
+@test 'ch-grow delete' {
+	run ch-grow build -t delete_test -f - . << 'EOF'
```

Comment:
A few suggestions here:

1. The convention for image names is hyphens, not underscores.
2. Use an image name that's more complex, so it tickles the code that converts image references to paths. Definitely you want a slash in it, but maybe other things too?
3. I recommend that before you build the image, you use `ch-grow list` to make sure it's *not* present.
4. If you don't care about the output, you can omit `run` along with echoing the output and checking the status. If you do that, output is captured automatically and test automatically fails if the command fails.